src/data/datasets/lits_seg_dataset.py

```diff
@@ -38,9 +38,9 @@ def __init__(self, data_split_file_path, transforms, augments=None, **kwargs):
             data_path
             for patient_dir in patient_dirs
             for data_path in zip(
-                sorted(patient_dir.glob('**/volume-*.nii')),
+                sorted(patient_dir.glob('**/*volume-*.nii')),
                 (
-                    sorted(patient_dir.glob('**/volume-*.nii'))
+                    sorted(patient_dir.glob('**/*volume-*.nii'))
                     if self.csv_name == 'testing.csv'
                     else sorted(patient_dir.glob('**/segmentation-*.nii'))
```

Comment:
Why need to add the "*" before "volume"?

Reply:
Because the filename of the official testing data is not start with 'volume'.

Reply:
I see, thanks.